README.md

```diff
@@ -1,32 +1,8 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
```

Comment:
- Hey, great job adding the readme as requested by the previous reviewer, however, let's update the readme to the current readme [template](https://github.com/microverseinc/readme-template) and customize it to suit your project needs by removing some headers that are not relevant to this current project, however, those headers would be relevant to the subsequent project as your make progress in the program. Also, kindly follow the structure of the template in order to be consistent to make your readme more professional.

Reply:
### PULL REQUEST TITLE

- Great job with the pull request title, however, the purpose of the project is to help you learn how to set up a professional Github repository and make a descriptive Pull Request with gives a summary of what you've done in a project.  You can further improve the Title by making it more descriptive e.g, `Hello Microverse Project`, as this is more descriptive as compared to `The New Pull Request `. Please, let's kindly update this.
---

### PULL REQUEST DESCRIPTION

- Great job so far setting up the `Hello Microverse Project`, however, you've not added any description that summarizes what you've done so far in the Pull Request.  The image below shows how to edit the Pull Request Description.

![Screenshot from 2022-12-05 16-32-11](https://user-images.githubusercontent.com/59397159/205679955-041b3086-0ca6-4380-bfa9-77cd502fe1fa.png)

- You can add something like 

```
**In this project milestone, I did the following:**

- Added the `index.html` file with an `h1` tag.
- Added the `style.css` file.
- Ensured there was no linter error.

```

This is just a starting point on how to have a descriptive summary in your Pull Request. Please, let's try and update this little issue to have our project approved. :pray: